custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/java_predictor.py

```diff
@@ -146,13 +146,10 @@ def supported_payload_formats(self):
         return formats
 
     def predict(self, **kwargs):
-        input_filename = kwargs.get(StructuredDtoKeys.FILENAME)
-        input_binary_data = kwargs.get(StructuredDtoKeys.BINARY_DATA)
-        if input_binary_data is not None:
-            input_binary_data = input_binary_data.decode("utf-8")
+        input_binary_data = kwargs.get(StructuredDtoKeys.BINARY_DATA).decode("utf-8")
 
         start_predict = time.time()
-        out_csv = self._predictor_via_py4j.predict(input_filename, input_binary_data)
+        out_csv = self._predictor_via_py4j.predict(None, input_binary_data)
```

Comment:
Does this call with 'None' imply that the java interface was not revised to comply with the new changes?

Reply:
Fixed java code